package/MDAnalysis/analysis/hole2/__init__.py

```diff
@@ -19,7 +19,207 @@
 # N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
 # MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
 # J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
-#
+
+
+"""HOLE Analysis --- :mod:`MDAnalysis.analysis.hole2.hole`
+=====================================================================================
+:Author: Lily Wang
+:Year: 2020
+:Copyright: GNU Public License v3
+.. versionadded:: 1.0.0
```

Comment:
```suggestion

.. versionadded:: 1.0.0
```

During the copy and paste you've removed many newlines that were necessary for displaying properly. For example, if I edit my `package/doc/sphinx/source/documentation_pages/analysis/hole2.rst`, I cannot build docs:

```
Warning, treated as error:
/Users/lily/pydev/mdanalysis/package/MDAnalysis/analysis/hole2/__init__.py:docstring of MDAnalysis.analysis.hole2:6:Field list ends without a blank line; unexpected unindent.
```

I would encourage you to build the documentation locally:

```
cd mdanalysis/package
python setup.py install
python setup.py build_sphinx
```

And keep editing and re-building until you get something similar to what it currently looks like in the docs. That's largely what's desired: https://docs.mdanalysis.org/stable/documentation_pages/analysis/hole2.html#hole-analysis-mdanalysis-analysis-hole2-hole

The spacing of the original text would be a good place to start.

Reply:
Sir ,i have done it now. please check it.